client/src/components/subscription/subscriptionListCard.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPen, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+const SubscriptionListCard = ({ details, setEditingSubscription, setActiveSubscription, handleDelete }) => {
+  const { name, nickname, dueDate, amount, subscriptionUuid } = details;
+
+  return (
+    <div className="w-100 p-2 d-flex justify-content-between align-items-center">
+      <button className="btn" type="button" onClick={() => setActiveSubscription(details)}>
+        <ul className="text-start">
```

Comment:
The text looks misaligned and provides too much white space on the left due to the padding from the `<ul>` element

```suggestion
        <ul className="text-start px-0">
```